### ISSUE TITLE
Live Demo cannot successfully load

### ISSUE THREAD

**Issue by visonpon** [vladmandic/human-motion#2]
Hi, @vladmandic thanks for sharing this wonderful work, but when i open the[ Live Demo](https://vladmandic.github.io/human-motion/src/index.html) and select the input, it has no response, hope you can check it ,thanks~

**Comment by vladmandic** [comment]
i've just tried the link and it works on my system  
what happens when you try it? do you see detection in lower left and just big right side is empty or?
can you open browser inspector and copy & paste the log output?

**Comment by visonpon** [comment]
there is no output in the right side like below:
![Snipaste_2022-06-22_09-36-04](https://user-images.githubusercontent.com/11517731/174924663-8bd362d6-889d-4316-9e83-597a8c8753db.png)

**Comment by vladmandic** [comment]
Like I said, please open browser inspector and copy&paste the log here.

**Comment by vladmandic** [comment]
closing temporarily as no required information was provided. issue can be reopened once more details are present.

**Comment by decacis** [comment]
Same problem. Here's the full console log:

```
Download the React DevTools for a better development experience: https://reactjs.org/link/react-devtools [index.js:244549:33](https://vladmandic.github.io/human-motion/dist/index.js)
Download the React DevTools for a better development experience: https://reactjs.org/link/react-devtools [index.js:307764:33](https://vladmandic.github.io/human-motion/dist/index.js)
17:38:01.454 Human: version: 2.11.1 [index.js:328622:13](https://vladmandic.github.io/human-motion/dist/index.js)
versions 
Object { human: "2.11.1", tfjs: "3.21.0", babylon: "5.28.0" }
[index.js:416808:32](https://vladmandic.github.io/human-motion/dist/index.js)
Babylon.js v5.28.0 - WebGL2 [index.js:3396:17](https://vladmandic.github.io/human-motion/dist/index.js)
enabled models: blazeface.json | facemesh.json [index.js:416808:32](https://vladmandic.github.io/human-motion/dist/index.js)
WebGL warning: generateMipmap: Tex image TEXTURE_CUBE_MAP_POSITIVE_X level 0 is incurring lazy initialization.
WebGL warning: generateMipmap: Tex image TEXTURE_CUBE_MAP_NEGATIVE_X level 0 is incurring lazy initialization.
WebGL warning: generateMipmap: Tex image TEXTURE_CUBE_MAP_POSITIVE_Y level 0 is incurring lazy initialization.
WebGL warning: generateMipmap: Tex image TEXTURE_CUBE_MAP_NEGATIVE_Y level 0 is incurring lazy initialization.
WebGL warning: generateMipmap: Tex image TEXTURE_CUBE_MAP_POSITIVE_Z level 0 is incurring lazy initialization.
WebGL warning: generateMipmap: Tex image TEXTURE_CUBE_MAP_NEGATIVE_Z level 0 is incurring lazy initialization.
17:38:02.221 Human: version: 2.11.1 [worker.js:54:13](https://vladmandic.github.io/human-motion/dist/worker.js)
human 
Object { worker: true, backend: "webgl", env: {…} }
[worker.js:87724:13](https://vladmandic.github.io/human-motion/dist/worker.js)
Source map error: Error: request failed with status 404
Resource URL: https://vladmandic.github.io/human-motion/dist/index.js
Source Map URL: index.js.map

video: ../assets/FaceModel.webm resolution: 720 x 720 duration: 89 [index.js:416808:32](https://vladmandic.github.io/human-motion/dist/index.js)
WebGL warning: clientWaitSync: ClientWaitSync must return TIMEOUT_EXPIRED until control has returned to the user agent's main loop. (only warns once)
ReferenceError: window is not defined
[worker.js:8557:9](https://vladmandic.github.io/human-motion/dist/worker.js)
WebGL warning: clientWaitSync: ClientWaitSync must return TIMEOUT_EXPIRED until control has returned to the user agent's main loop. (only warns once)
Source map error: Error: request failed with status 404
Resource URL: https://vladmandic.github.io/human-motion/dist/index.js
Source Map URL: index.js.map
```

After selecting the "Sample: Face" option, it loads the video and after a couple of seconds, it outputs "ReferenceError: window is not defined".

My system: Firefox 106.0.5 (64-bit) Windows 10 Pro

Also tested in Chrome 107.0.5304.107 and Firefox in private mode with no extensions.

**Comment by vladmandic** [comment]
its not the same problem - don't append to issue that was closed 4 months ago  
if original issue is closed, please open a new issue instead  
  
anyhow, your specific problem was due to regression bug in `tfjs` **3.21.0** (it was ok with 3.20.0) and it was corrected in 3.21.1, but i didn't rebuild `human-motion`  

second issue is that **firefox** (still) does not support `offscreenCanvas`, and this demo uses webworkers by default, so it wouldn't work in firefox. i've added detection code so if `offscreenCanvas` functionality is not present, it will run in main thread instead of web worker.

anyhow, its rebuilt now and error should be gone.
